onnxruntime/core/graph/contrib_ops/contrib_defs.cc

```diff
@@ -309,10 +309,46 @@ mask_index shall not be provided.)DOC";
       .Input(1, "weight", "2D input tensor with shape (hidden_size, 3 * hidden_size)", "T")
       .Input(2, "bias", "1D input tensor with shape (3 * hidden_size)", "T")
       .Input(3, "mask_index", "Attention mask index with shape (batch_size).", "M", OpSchema::Optional)
-      .Output(0, "output", "3D output tensor with shape (batch_size, sequence_length, hidden_size)", "T")
+      .Input(4, "past", "past state for key and value with shape (2, batch_size, num_heads, past_sequence_length, head_size).", "T", OpSchema::Optional)
```

Comment:
>2 [](start = 66, length = 1)

How about split the past state for k and v

Reply:
The export onnx model has such format